environments/pull-environment.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
I found it much easier to have a different shell file for each `amplify pull` execution, to go around side effects like `cd`ing into directories.